polylogue/html.py

```
@@ -314,14 +460,34 @@ def _fmt(value: object) -> object:
+            "kind": "image" if _attachment_preview_src(att) else "file",
+            "icon": _attachment_icon(att),
         }
-        for att in document.attachments
+        for att in attachments_sorted
     ]
+    for att in attachments_sorted:
+        if getattr(att, "size_bytes", None):
+            try:
+                attachment_total_bytes += int(att.size_bytes or 0)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)